core/blockchain/impl/storage_util.hpp

```diff
@@ -9,7 +9,24 @@
 #include "common/buffer.hpp"
 #include "primitives/block_header.hpp"
 #include "primitives/block_id.hpp"
-#include "storage/buffer_map_types.hpp"
+#include "storage/spaced_storage.hpp"
+
+/**
+ * Storage schema overview
+ *
+ * A key-value approach is used for block storage.
+ * Different parts containing a block are stored in multiple storage spaces but
+ * have to be addressed with the same key.
+ *
+ * A key is the combination of block's number concatenated with its hash
+ * resulting in a 512-bit byte buffer. Let's name it as NumHashKey.
```

Comment:
minor: 32 (number) + 256 (hash) != 512

Reply:
32*8=256
256+256=512
Is there something I am missing?

Reply:
Thanks! Fixed